README.rst

```diff
@@ -12,8 +12,8 @@ Introduction
     :alt: Discord
 
 
-.. image:: https://github.com/circuitpython/CircuitPython_DisplayIO_Annotation/workflows/Build%20CI/badge.svg
-    :target: https://github.com/circuitpython/CircuitPython_DisplayIO_Annotation/actions
+.. image:: https://github.com/circuitpython/CircuitPython_Org_DisplayIO_Annotation/workflows/Build%20CI/badge.svg
+    :target: https://github.com/circuitpython/CircuitPython_Org_DisplayIO_Annotation/actions
     :alt: Build Status
 
 
```

Comment:
At line 37 below, a newline is needed before 'Installing from PyPI'
At line 42 below, remove the `adafruit-`

Reply:
Forgot to check in on this back when it happened. Thanks for looking this over and catching those!